tests/helpers.py

```diff
@@ -31,13 +31,38 @@ def make_env_tiny(bf16_enable=True):
   return env, config
 
 
+def make_mixtral_env(bf16_enable=True):
+  torch_dtype = torch.bfloat16 if bf16_enable else torch.float32
+  torch.set_default_dtype(torch_dtype)
+  jax.config.update("jax_dynamic_shapes", False)
+  jax.config.update("jax_traceback_filtering", "off")
+  config = mixtral_config.ModelArgs.from_name("Mixtral-tiny")
+  environment_data = environment.JetEngineEnvironmentData()
+  environment_data.max_input_sequence_length = 128
+  environment_data.cache_sequence_length = 128
+  environment_data.bf16_enable = bf16_enable
+  environment_data.model_type = "mixtral"
+  environment_data.batch_size = 1
+  environment_data.num_layers = config.n_layer
+  environment_data.cache_shape = (
+      1,
+      config.n_local_heads,
+      environment_data.cache_sequence_length,
+      config.dim // config.n_head,
+  )
+  env = environment.JetEngineEnvironment(environment_data)
+  env.apply_sharding = lambda *args, **kwargs: None  # don't shard on cpu
+  return env, config
+
+
 def to_xla_tensor(tree):
   return torch_xla2.default_env().to_xla(tree)
 
 
 def call_xla_model(model, weights, args):
   with jax.default_device(jax.devices("cpu")[0]):
-    xla_weights, xla_inputs = to_xla_tensor((weights, args))
-    result = torch.func.functional_call(model, xla_weights, xla_inputs)
-    result_torch = torch_xla2.tensor.j2t(result._elem)
-    return result_torch
+    with torch_xla2.default_env():
+      xla_weights, xla_inputs = to_xla_tensor((weights, args))
+      result = torch.func.functional_call(model, xla_weights, xla_inputs)
```

Comment:
only put this line inside of `torch_xla2.default_env()` put others outside of it.

Reply:
the to_xla_tensor should be outside of torch_xla2.default_env